src/utils/index.ts

```diff
@@ -14,6 +14,8 @@ export function renderConfigToUrlParams(config: StacRenderObject): string {
       queryObj[key] = `${value}`;
     }
   }
+  queryObj['bands'] = queryObj['assets'] 
+  delete queryObj['assets']
```

Comment:
Looking at the types, a renders object now has `assets` and optional `bands` — is this something we'll consolidate in the render extension at some point, ie. only use `bands` instead of `assets`?

Reply:
I think we need to resolve this with @vincentsarago and @sharkinsspatial in https://github.com/developmentseed/stac-explorer/pull/30/files#r1512742976.

Right now the titiler-cmr API only accepts bands, so we need to make that translation, but there was a suggestion that the API changes to use assets.